portfolio/src/main/java/com/google/sps/servlets/contactFormServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.sps.Contact;
+
+@WebServlet("/contact-form-handler")
+public class contactFormServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String contactMessage = request.getParameter("message");
+    String contactName = request.getParameter("name");
+    String contactEmail = request.getParameter("email");
+
+    Contact contact = new Contact(contactName, contactEmail, contactMessage);
```

Comment:
What is being done with contact? Seems like you are creating an object that is never used.